python/ola/ClientWrapperTest.py

```diff
@@ -181,19 +182,32 @@ def d():
     self.start = datetime.datetime.now()
     wrapper.Run()
 
+    # Everything has been called
     self.assertIsNotNone(results.a_called)
     self.assertIsNotNone(results.b_called)
     self.assertIsNotNone(results.c_called)
     self.assertIsNotNone(results.d_called)
 
-    self.assertTrue(results.a_called - self.start <
-                    datetime.timedelta(milliseconds=5))
-    self.assertTrue(results.b_called - self.start >=
-                    datetime.timedelta(milliseconds=5))
-    self.assertTrue(results.c_called - self.start >=
-                    datetime.timedelta(milliseconds=10))
-    self.assertTrue(results.d_called - self.start >=
-                    datetime.timedelta(milliseconds=15))
+    # Check when the callbacks were called. Allow 500 microseconds of drift.
+    # Called immediately
+    a_diff = results.a_called - self.start
+    self.assertAlmostEqual(a_diff, datetime.timedelta(microseconds=0),
```

Comment:
While it's no technical change to the output, this will avoid any typos or mistakes in future!
```suggestion
    self.assertAlmostEqual(a_diff, datetime.timedelta(microseconds=0),
```

Reply:
Not sure what you're asking?  Both lines appear the same.

Reply:
:facepalm: Edited!